undefined reference to `esp_eth_phy_new_jl1101'

### The problem

After updating to the new `2023.02` release this morning, I attempted to update the firmware on my BTLE proxy device and got this failure during compilation:

> 
> /home/karl/.platformio/packages/toolchain-xtensa-esp32@8.4.0+2021r2-patch3/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: .pioenvs/remote-bt-01/src/esphome/components/ethernet/ethernet_component.cpp.o: in function `esphome::ethernet::EthernetComponent::setup()':
> /home/karl/Projects/esphome/hardware/build-root/remote-bt-01/src/esphome/components/ethernet/ethernet_component.cpp:100: undefined reference to `esp_eth_phy_new_jl1101'


### Which version of ESPHome has the issue?

2023.02

### What type of installation are you using?

pip

### Which version of Home Assistant has the issue?

N/A

### What platform are you using?

ESP32

### Board

LILYGO TTGO T-Internet-POE ESP32-WROOM LAN8720A

### Component causing the issue

ethernet

### Example YAML snippet

```yaml
ethernet:
  type: LAN8720
  mdc_pin: GPIO23
  mdio_pin: GPIO18
  clk_mode: GPIO17_OUT
  phy_addr: 0
```


### Anything in the logs that might be useful for us?

_No response_

### Additional information

_No response_

Same here.

```
Linking .pioenvs/zb-gw03-coordinator/bootloader.elf
Building .pioenvs/zb-gw03-coordinator/bootloader.bin
Creating esp32 image...
Successfully created esp32 image.
Linking .pioenvs/zb-gw03-coordinator/firmware.elf
/home/runner/.platformio/packages/toolchain-xtensa-esp32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: .pioenvs/zb-gw03-coordinator/src/esphome/components/ethernet/ethernet_component.o:(.literal._ZN7esphome8ethernet17EthernetComponent5setupEv+0x44): undefined reference to `esp_eth_phy_new_jl1101'
/home/runner/.platformio/packages/toolchain-xtensa-esp32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: .pioenvs/zb-gw03-coordinator/src/esphome/components/ethernet/ethernet_component.o: in function `esphome::ethernet::EthernetComponent::setup()':
/home/runner/work/esphome-zb-gw03/esphome-zb-gw03/.esphome/build/zb-gw03-coordinator/src/esphome/components/ethernet/ethernet_component.cpp:57: undefined reference to `esp_eth_phy_new_jl1101'
collect2: error: ld returned 1 exit status
*** [.pioenvs/zb-gw03-coordinator/firmware.elf] Error 1
========================= [FAILED] Took 151.86 seconds =========================
Error: Process completed with exit code 1.
```

https://github.com/syssi/esphome-zb-gw03/actions/runs/4188369897/jobs/7259417281

Can be reproduced using this example configuration:

https://raw.githubusercontent.com/syssi/esphome-zb-gw03/main/coordinator-example.yaml

I clean build doesn't help. I've removed the cache of the CI pipeline manually and tried a local build:

```
$ cd /tmp
$ git clone git@github.com:syssi/esphome-zb-gw03.git 
$ cd esphome-zb-gw03
$ esphome run coordinator-example.yaml
[...]
Linking .pioenvs/zb-gw03-coordinator/bootloader.elf
Building .pioenvs/zb-gw03-coordinator/bootloader.bin
Creating esp32 image...
Successfully created esp32 image.
Linking .pioenvs/zb-gw03-coordinator/firmware.elf
/home/sebastian/.platformio/packages/toolchain-xtensa-esp32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: .pioenvs/zb-gw03-coordinator/src/esphome/components/ethernet/ethernet_component.o:(.literal._ZN7esphome8ethernet17EthernetComponent5setupEv+0x44): undefined reference to `esp_eth_phy_new_jl1101'
/home/sebastian/.platformio/packages/toolchain-xtensa-esp32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: .pioenvs/zb-gw03-coordinator/src/esphome/components/ethernet/ethernet_component.o: in function `esphome::ethernet::EthernetComponent::setup()':
/tmp/esphome-zb-gw03/.esphome/build/zb-gw03-coordinator/src/esphome/components/ethernet/ethernet_component.cpp:57: undefined reference to `esp_eth_phy_new_jl1101'
collect2: error: ld returned 1 exit status
*** [.pioenvs/zb-gw03-coordinator/firmware.elf] Error 1
$
```

> @kquinsland Can you try a clean build? (But also try 2023.2.1 first as that should run a clean build automatically)

I failed to mention in the first post that I explicitly set the `build_path` to a new value before attempting to build.

Here's my shell history immediately after `pip install --upgrade esphome` and trying to build:

```shell
# This is a build attempt before cleaning....
Archiving .pioenvs/remote-bt-01/libFrameworkArduino.a
Indexing .pioenvs/remote-bt-01/libFrameworkArduino.a
Linking .pioenvs/remote-bt-01/firmware.elf
/home/karl/.platformio/packages/toolchain-xtensa-esp32@8.4.0+2021r2-patch3/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: .pioenvs/remote-bt-01/src/esphome/components/ethernet/ethernet_component.cpp.o:(.literal._ZN7esphome8ethernet17EthernetComponent5setupEv+0x44): undefined reference to `esp_eth_phy_new_jl1101'
/home/karl/.platformio/packages/toolchain-xtensa-esp32@8.4.0+2021r2-patch3/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: .pioenvs/remote-bt-01/src/esphome/components/ethernet/ethernet_component.cpp.o: in function `esphome::ethernet::EthernetComponent::setup()':
/home/karl/Projects/esphome/hardware/build-root/remote-bt-01/src/esphome/components/ethernet/ethernet_component.cpp:100: undefined reference to `esp_eth_phy_new_jl1101'
collect2: error: ld returned 1 exit status
*** [.pioenvs/remote-bt-01/firmware.elf] Error 1
======================================================================= [FAILED] Took 12.17 seconds =======================================================================
❯ rm -rf /home/karl/.platformio/packages/toolchain-xtensa-esp32*
❯ esphome clean projects/ha_bt_proxy.yaml
INFO Reading configuration projects/ha_bt_proxy.yaml...
INFO Deleting projects/../build-root/remote-bt-01/.pioenvs
INFO Deleting projects/../build-root/remote-bt-01/.piolibdeps
INFO Done!
```


Here's the build post `clean`:

```shell
❯ esphome -s version_hash $(git rev-parse --short HEAD) run projects/ha_bt_proxy.yaml
INFO Reading configuration projects/ha_bt_proxy.yaml...
INFO Generating C++ source...
INFO Compiling app...
Processing remote-bt-01 (board: esp32dev; framework: arduino; platform: platformio/espressif32 @ 5.2.0)
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Tool Manager: Installing espressif/toolchain-xtensa-esp32 @ 8.4.0+2021r2-patch3
INFO Installing espressif/toolchain-xtensa-esp32 @ 8.4.0+2021r2-patch3
Downloading  [####################################]  100%          
Unpacking  [####################################]  100%          
Tool Manager: toolchain-xtensa-esp32@8.4.0+2021r2-patch3 has been installed!
INFO toolchain-xtensa-esp32@8.4.0+2021r2-patch3 has been installed!
Library Manager: Installing esphome/AsyncTCP-esphome @ 1.2.2
INFO Installing esphome/AsyncTCP-esphome @ 1.2.2
Unpacking  [####################################]  100%
Library Manager: AsyncTCP-esphome@1.2.2 has been installed!
INFO AsyncTCP-esphome@1.2.2 has been installed!
Library Manager: Installing esphome/ESPAsyncWebServer-esphome @ 2.1.0
INFO Installing esphome/ESPAsyncWebServer-esphome @ 2.1.0
Unpacking  [####################################]  100%
Library Manager: ESPAsyncWebServer-esphome@2.1.0 has been installed!
INFO ESPAsyncWebServer-esphome@2.1.0 has been installed!
Library Manager: Resolving dependencies...
INFO Resolving dependencies...
Library Manager: Installing bblanchon/ArduinoJson @ 6.18.5
INFO Installing bblanchon/ArduinoJson @ 6.18.5
Unpacking  [####################################]  100%
Library Manager: ArduinoJson@6.18.5 has been installed!
INFO ArduinoJson@6.18.5 has been installed!
HARDWARE: ESP32 240MHz, 320KB RAM, 4MB Flash
 - toolchain-xtensa-esp32 @ 8.4.0+2021r2-patch3
LDF: Library Dependency Finder -> https://bit.ly/configure-pio-ldf
Dependency Graph
|-- AsyncTCP-esphome @ 1.2.2
|-- WiFi @ 2.0.0
|-- FS @ 2.0.0
|-- Update @ 2.0.0
|-- ESPAsyncWebServer-esphome @ 2.1.0
|   |-- AsyncTCP-esphome @ 1.2.2
|-- ESPmDNS @ 2.0.0
|-- ArduinoJson @ 6.18.5
Compiling .pioenvs/remote-bt-01/src/esphome/components/api/api_connection.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/api/api_frame_helper.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/api/api_pb2.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/api/api_pb2_service.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/api/api_server.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/api/list_entities.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/api/proto.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/api/subscribe_state.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/api/user_services.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/atc_mithermometer/atc_mithermometer.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/binary_sensor/automation.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/binary_sensor/binary_sensor.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/binary_sensor/filter.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/bluetooth_proxy/bluetooth_connection.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/bluetooth_proxy/bluetooth_proxy.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/button/button.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/esp32/core.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/esp32/gpio.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/esp32/preferences.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/esp32_ble/ble.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/esp32_ble/ble_advertising.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/esp32_ble/ble_uuid.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/esp32_ble_client/ble_characteristic.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/esp32_ble_client/ble_client_base.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/esp32_ble_client/ble_service.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/esp32_ble_tracker/esp32_ble_tracker.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/ethernet/ethernet_component.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/json/json_util.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/logger/logger.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/md5/md5.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/mdns/mdns_component.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/mdns/mdns_esp32.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/mdns/mdns_esp8266.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/mdns/mdns_rp2040.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/network/util.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/ota/ota_backend_arduino_esp32.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/ota/ota_backend_arduino_esp8266.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/ota/ota_backend_arduino_rp2040.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/ota/ota_backend_esp_idf.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/ota/ota_component.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/prometheus/prometheus_handler.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/restart/button/restart_button.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/safe_mode/button/safe_mode_button.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/sensor/automation.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/sensor/filter.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/sensor/sensor.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/sntp/sntp_component.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/socket/bsd_sockets_impl.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/socket/lwip_raw_tcp_impl.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/socket/socket.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/status/status_binary_sensor.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/template/sensor/template_sensor.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/time/automation.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/time/real_time_clock.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/uptime/uptime_sensor.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/web_server/list_entities.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/web_server/web_server.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/web_server_base/web_server_base.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/xiaomi_ble/xiaomi_ble.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/components/xiaomi_lywsd02/xiaomi_lywsd02.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/core/application.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/core/color.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/core/component.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/core/component_iterator.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/core/controller.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/core/entity_base.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/core/helpers.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/core/log.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/core/scheduler.cpp.o
Compiling .pioenvs/remote-bt-01/src/esphome/core/util.cpp.o
Compiling .pioenvs/remote-bt-01/src/main.cpp.o
Building .pioenvs/remote-bt-01/bootloader.bin
Generating partitions .pioenvs/remote-bt-01/partitions.bin
Creating esp32 image...
Successfully created esp32 image.
Compiling .pioenvs/remote-bt-01/libc06/AsyncTCP-esphome/AsyncTCP.cpp.o
Compiling .pioenvs/remote-bt-01/lib2f8/WiFi/WiFi.cpp.o
Compiling .pioenvs/remote-bt-01/lib2f8/WiFi/WiFiAP.cpp.o
Compiling .pioenvs/remote-bt-01/lib2f8/WiFi/WiFiClient.cpp.o
Compiling .pioenvs/remote-bt-01/lib2f8/WiFi/WiFiGeneric.cpp.o
Compiling .pioenvs/remote-bt-01/lib2f8/WiFi/WiFiMulti.cpp.o
Compiling .pioenvs/remote-bt-01/lib2f8/WiFi/WiFiSTA.cpp.o
Compiling .pioenvs/remote-bt-01/lib2f8/WiFi/WiFiScan.cpp.o
Compiling .pioenvs/remote-bt-01/lib2f8/WiFi/WiFiServer.cpp.o
Compiling .pioenvs/remote-bt-01/lib2f8/WiFi/WiFiUdp.cpp.o
Compiling .pioenvs/remote-bt-01/lib203/FS/FS.cpp.o
Compiling .pioenvs/remote-bt-01/lib203/FS/vfs_api.cpp.o
Compiling .pioenvs/remote-bt-01/lib580/Update/HttpsOTAUpdate.cpp.o
Compiling .pioenvs/remote-bt-01/lib580/Update/Updater.cpp.o
Compiling .pioenvs/remote-bt-01/lib98e/ESPAsyncWebServer-esphome/AsyncEventSource.cpp.o
Compiling .pioenvs/remote-bt-01/lib98e/ESPAsyncWebServer-esphome/AsyncWebSocket.cpp.o
Compiling .pioenvs/remote-bt-01/lib98e/ESPAsyncWebServer-esphome/WebAuthentication.cpp.o
Compiling .pioenvs/remote-bt-01/lib98e/ESPAsyncWebServer-esphome/WebHandlers.cpp.o
Compiling .pioenvs/remote-bt-01/lib98e/ESPAsyncWebServer-esphome/WebRequest.cpp.o
Archiving .pioenvs/remote-bt-01/lib2f8/libWiFi.a
Archiving .pioenvs/remote-bt-01/libc06/libAsyncTCP-esphome.a
Indexing .pioenvs/remote-bt-01/libc06/libAsyncTCP-esphome.a
Indexing .pioenvs/remote-bt-01/lib2f8/libWiFi.a
Compiling .pioenvs/remote-bt-01/lib98e/ESPAsyncWebServer-esphome/WebResponses.cpp.o
Compiling .pioenvs/remote-bt-01/lib98e/ESPAsyncWebServer-esphome/WebServer.cpp.o
Compiling .pioenvs/remote-bt-01/lib981/ESPmDNS/ESPmDNS.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/Esp.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/FirmwareMSC.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/FunctionalInterrupt.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/HWCDC.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/HardwareSerial.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/IPAddress.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/IPv6Address.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/MD5Builder.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/Print.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/Stream.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/StreamString.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/Tone.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/USB.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/USBCDC.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/USBMSC.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/WMath.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/WString.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/base64.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/cbuf.cpp.o
Archiving .pioenvs/remote-bt-01/lib981/libESPmDNS.a
Archiving .pioenvs/remote-bt-01/lib580/libUpdate.a
Archiving .pioenvs/remote-bt-01/lib203/libFS.a
Indexing .pioenvs/remote-bt-01/lib981/libESPmDNS.a
Indexing .pioenvs/remote-bt-01/lib580/libUpdate.a
Indexing .pioenvs/remote-bt-01/lib203/libFS.a
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-adc.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-bt.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-cpu.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-dac.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-gpio.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-i2c-slave.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-i2c.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-ledc.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-matrix.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-misc.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-psram.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-rgb-led.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-rmt.c.o
Archiving .pioenvs/remote-bt-01/lib98e/libESPAsyncWebServer-esphome.a
Indexing .pioenvs/remote-bt-01/lib98e/libESPAsyncWebServer-esphome.a
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-sigmadelta.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-spi.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-time.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-timer.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-tinyusb.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-touch.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/esp32-hal-uart.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/firmware_msc_fat.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/libb64/cdecode.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/libb64/cencode.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/main.cpp.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/stdlib_noniso.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/wiring_pulse.c.o
Compiling .pioenvs/remote-bt-01/FrameworkArduino/wiring_shift.c.o
Archiving .pioenvs/remote-bt-01/libFrameworkArduino.a
Indexing .pioenvs/remote-bt-01/libFrameworkArduino.a
Linking .pioenvs/remote-bt-01/firmware.elf
/home/karl/.platformio/packages/toolchain-xtensa-esp32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: .pioenvs/remote-bt-01/src/esphome/components/ethernet/ethernet_component.cpp.o:(.literal._ZN7esphome8ethernet17EthernetComponent5setupEv+0x44): undefined reference to `esp_eth_phy_new_jl1101'
/home/karl/.platformio/packages/toolchain-xtensa-esp32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: .pioenvs/remote-bt-01/src/esphome/components/ethernet/ethernet_component.cpp.o: in function `esphome::ethernet::EthernetComponent::setup()':
/home/karl/Projects/esphome/hardware/build-root/remote-bt-01/src/esphome/components/ethernet/ethernet_component.cpp:100: undefined reference to `esp_eth_phy_new_jl1101'
collect2: error: ld returned 1 exit status
*** [.pioenvs/remote-bt-01/firmware.elf] Error 1
```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,8 +1,6 @@
 include LICENSE
 include README.md
 include requirements.txt
-include esphome/dashboard/templates/*.html
-recursive-include esphome/dashboard/static *.ico *.js *.css *.woff* LICENSE
-recursive-include esphome *.cpp *.h *.tcc
+recursive-include esphome *.cpp *.h *.tcc *.c
 recursive-include esphome *.py.script
 recursive-include esphome LICENSE.txt

